user_store_exercise/src/main.rs

```diff
@@ -1,15 +1,30 @@
+use std::sync::{Arc, Mutex};
+
 use app::App;
-use libs::{
-    console::ctc,
-    db::Db,
-};
+use libs::db::Db;
 
 mod app;
 mod libs;
 mod models;
 
 fn main() {
     let db = Db::new();
-    let mut app = App::new(db);
-    app.run();
+    let app = Arc::new(Mutex::new(App::new(db)));
```

Comment:
I was trying just to do a small Box here to pass it, and it works until is `mut` when it is mut you need an `Arc` and a `Mutex`

I posted the small code example in [here]https://gist.github.com/vikkio88/ce94e6ba4b907f1d33f3a345964314ee()  just to avoid you to clone all this shit